core/src/test/scala/com/dimafeng/testcontainers/BaseSpec.scala

```diff
@@ -5,11 +5,12 @@ import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatestplus.mockito.MockitoSugar
+import scala.reflect.ClassTag
 
-abstract class BaseSpec[T: Manifest]
+abstract class BaseSpec[T: ClassTag]
```

Comment:
this breaks binary compat

Reply:
It's in tests, so I don't think that this is a problem.